cypress/local-only/tests/integration/caseDetail/caseInformation/edit-case-detail.cy.ts

```diff
@@ -0,0 +1,327 @@
+import { loginAsDocketClerk } from 'cypress/helpers/authentication/login-as-helpers';
```

Comment:
We can add more to this file since it currently just tests renaming a case caption, changing the case type, and statistics-related things. I thought it more efficient to create the case once and do multiple case-detail edits rather than having separate files that each create a case.